### ISSUE TITLE
is it possible to connect yi action cam to server/vps?

### ISSUE THREAD

**Issue by sprabowo** [mariomka/yi-action-camera#3]

**Comment by mariomka** [comment]
The connection with the camera is via WiFi, maybe it is possible with a VPN or similar.

**Comment by sprabowo** [comment]
yeah its possibly to connect camera with wiFi, but if i wanna make it accessible on site by using ngrok. it need LAN connection
